services/telemetry/telemetry.go

```diff
@@ -450,6 +450,7 @@ func (ts *TelemetryService) trackConfig() {
 		"restrict_link_previews":                                  isDefault(*cfg.ServiceSettings.RestrictLinkPreviews, ""),
 		"enable_custom_groups":                                    *cfg.ServiceSettings.EnableCustomGroups,
 		"post_priority":                                           *cfg.ServiceSettings.PostPriority,
+		"self_hosted_first_time_purchase":                         *cfg.ServiceSettings.SelfHostedFirstTimePurchase,
```

Comment:
nit: given that this will expand to more than just first time purchase, can we generalize to:

```suggestion
		"self_hosted_purchase":                         *cfg.ServiceSettings.SelfHostedPurchase,
```

Reply:
I like the shorter, clearer name. Will add to the next round of PRs.